runtime/optwin.vim

```diff
@@ -230,14 +218,6 @@ call <SID>AddOption("compatible", gettext("behave very Vi compatible (not advisa
 call <SID>BinOptionG("cp", &cp)
 call <SID>AddOption("cpoptions", gettext("list of flags to specify Vi compatibility"))
 call <SID>OptionG("cpo", &cpo)
-call <SID>AddOption("paste", gettext("paste mode, insert typed text literally"))
-call <SID>BinOptionG("paste", &paste)
```

Comment:
Do we keep deprecated options in optwin?

Reply:
i'd lean towards leaving it in, no strong opinion